README.md

```diff
@@ -1 +1,28 @@
-# eq-smoke-tests
\ No newline at end of file
+# eq-smoke-tests
+
+## Background
+
+This repo is a simple repeatable smoke test that verifies the eQ services work together
+So that when new versions of service are deployed we can be sure we haven't broken anything.
+
+## PREREQUISITE
+Oracle JDK v1.8+
+You can check your Java version in the terminal with java -version (single dash)
+
+Be sure to install the JDK not the JRE.
+
+## Installation
+```
+npm install chimp
```

Comment:
Given that we have a `package.json` should this not just be `npm install` or `yarn install` if we are using yarn?